game_manager.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'game'
+require_relative 'author'
+require 'json'
+require_relative './data_storage_handler'
+
+class GameManager
+  attr_accessor :games
+
+  def initialize
+    @games = []
+  end
+
+  def add_game(author_manager)
+    puts 'Multiplayer? (y/n)'
+    multi_player = gets.chomp
+    puts 'Last played at (yyyy-mm-dd)'
+    last_played_at = gets.chomp
+    puts 'Publish date (yyyy-mm-dd)'
+    publish_date = gets.chomp
+    game = Game.new(multi_player: multi_player, last_played_at: last_played_at, publish_date: publish_date)
+    game.move_to_archive? if game.can_be_archived?
+    puts 'Would you like to add an author to the agame? [Y/N]?'
+    answer = gets.chomp
+    author_manager.add_author if answer.downcase == 'y'
```

Comment:
[Optional]
Maybe we can give him a list of authors to choose from or he can add a new author.
You can check MusicAlbum Manager Class for reference.